packages/@aws-cdk/aws-appmesh/lib/virtual-node-listener.ts

```diff
@@ -157,13 +165,24 @@ export abstract class VirtualNodeListener {
 
 }
 
+/**
+ * Renders the TLS config for a listener
+ */
+function renderTls(scope: cdk.Construct, tlsCertificate: TlsCertificate): CfnVirtualNode.ListenerTlsProperty {
+  return {
+    certificate: tlsCertificate.bind(scope).tlsCertificate,
+    mode: tlsCertificate.tlsMode.toString(),
+  };
+}
+
```

Comment:
Please move this function down, next to the other private functions in this file.

Maybe make this a `private` function of `VirtualNodeListenerImpl`, like `renderHealthCheck` is now?